db/index.js

```diff
@@ -90,7 +91,7 @@ Article.belongsToMany(Category, {through: 'ArticleCategory'});
 
 ///// USE THIS TO SEED DB ///////
 
-// sequelize.sync({ force: true }).then(async () => {
+sequelize.sync({ force: true }).then(() => { console.log('db synced')});
```

Comment:
is this sync supposed to be uncommented? @michellelockett

Reply:
noooooooope